Formula/bookkeeper.rb

```diff
@@ -0,0 +1,49 @@
+class Bookkeeper < Formula
+  desc "Replicated log service for building replicated state machines"
+  homepage "https://bookkeeper.apache.org/"
+  url "https://apache.claz.org/bookkeeper/bookkeeper-4.5.0/bookkeeper-server-4.5.0-bin.tar.gz"
+  sha256 "757abd3083291cb28d6a40342ac6e34b3b802ecc36ba5487a08df92bccc9768e"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/bookkeeper"]
+  end
+
+  plist_options :manual => "bookeeper localbookie 4"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <false/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/bookkeeper</string>
+          <string>localbookie</string>
+          <string>4</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{var}</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/bookkeeper.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/bookkeeper.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_match "Usage: bookkeeper", shell_output("#{bin}/bookkeeper help")
```

Comment:
See if you can replace this with a test that checks more of the functionality.  Thanks!

Reply:
it would be difficult to do since this generally serves as a component of distributed system, it will create a significant number of artifacts once started like default namespaces etc, which would need to cleaned up after the tests is done. I would avoid that.

Reply:
The test is run in a temporary directory, so that should be fine.

Reply:
added extended test